### PR TITLE
log filtered cloudprofiles

### PR DESCRIPTION
**Pull request by grolu**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```improvement user

```

### CODE REVIEW

File: backend/lib/services/cloudprofiles.js [hunk 1]
```diff
@@ -68,7 +68,13 @@ exports.list = function () {
...
     .chain(cloudProfiles)
     .map(assignSeedsToCloudProfileIteratee(seeds))
-    .filter('data.seeds')
+    .filter(cloudProfile => {
+      if (!_.isEmpty(cloudProfile.data.seeds)) {
+        return true
+      }
+      console.log(`No matching seed for cloud profile with name ${cloudProfile.metadata.name} found`)
+      return false
+    })
```
Comment (holgerkoser) [inline backend/lib/services/cloudprofiles.js:77]: .filter('data.seeds.length')

**Comment by grolu** [comment]
@holgerkoser the point of this PR to (re)add the missing logging for the case where cloud profiles without seeds are removed from the list. This makes it easier to identify why an infrastructure / cloud profile is missing in the frontend.

File: backend/lib/services/cloudprofiles.js [hunk 2]
```diff
@@ -68,7 +68,13 @@ exports.list = function () {
   return _
     .chain(cloudProfiles)
     .map(assignSeedsToCloudProfileIteratee(seeds))
-    .filter('data.seeds')
+    .filter(cloudProfile => {
+      if (!_.isEmpty(cloudProfile.data.seeds)) {
+        return true
+      }
+      console.log(`No matching seed for cloud profile with name ${cloudProfile.metadata.name} found`)
```
Comment (petersutter) [inline backend/lib/services/cloudprofiles.js:75]: use logger

Comment (grolu) [reply backend/lib/services/cloudprofiles.js]: done

**Pull request merged**